### PR TITLE
Implement driving distance API call

### PR DESCRIPTION
**Pull request by marcelpuyat**

### CODE REVIEW

File: GeoConnector.py [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
 		self.client = googlemaps.Client(service_key)
 
-	def get_driving_time(self, start_pt, end_pt):
-		return 0
-
-	def get_walking_dist(self, start_pt, end_pt):
-		'''Get the walking distance from start_pt to end_pt.
-		If walking between the two points is not possible, None is returned.
+	def get_dist(self, start_pt, end_pt, mode):
+		'''Get the distance from start_pt to end_pt. Mode must either be WALKING_MODE
```
Comment (arthurbrant21) [inline GeoConnector.py]: I think you have too many tabs here.

File: GeoConnector.py [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+		or DRIVING_MODE. If traveling between the two points is not possible, None is returned.
```
Comment (arthurbrant21) [inline GeoConnector.py]: next lines should be indented.

File: GeoConnector.py [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
 		Returns a 2 element tuple where the first element is the travel duration in
 		minutes and the 2nd is the distance in miles.
 		'''
-		distance_matrix = self.client.distance_matrix(start_pt, end_pt, self.WALKING_MODE)
+		if mode != self.WALKING_MODE and mode != self.DRIVING_MODE:
+			print >> sys.stderr, 'Invalid mode: ' + mode
```
Comment (arthurbrant21) [inline GeoConnector.py]: I"m happy just to use print. Presumably, when we move past the mvp we will setup actual logging.

File: GeoConnector.py (hunk 1 above)
Comment (marcelpuyat) [reply GeoConnector.py]: done

File: GeoConnector.py (hunk 2 above)
Comment (marcelpuyat) [reply GeoConnector.py]: done

File: GeoConnector.py (hunk 3 above)
Comment (marcelpuyat) [reply GeoConnector.py]: done

**Pull request merged**